Cargo.toml

```diff
@@ -15,9 +15,9 @@ rust-version = "1.60.0"
 [dependencies]
 async-trait = "0.1.56"
 catty = "0.1.5"
-futures-core = { version = "0.3.21", default-features = false, features = ["alloc"] }
+futures-core = "0.3.21"
```

Comment:
Does futures core have no default features?

Reply:
It does: `alloc` :)